pkg/driver/nodeserver.go

```diff
@@ -126,15 +126,15 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 
 	mntPoint, err := util.IsLikelyMountPoint(targetPath)
 	if os.IsNotExist(err) {
-		if err = os.MkdirAll(targetPath, 0700); err != nil {
+		if err = os.MkdirAll(targetPath, (0700 | os.ModeSticky)); err != nil {
```

Comment:
Ditto. Please explain the use of ModeSticky here and consider whether it makes sense on a owner-only-writable mount point.